onboard-iOS/DesignSystem/Values/TextStyle.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  TextStyle.swift
+//  onboard-iOS
+//
+//  Created by 혜리 on 2023/10/08.
+//
+
+import UIKit
+
+struct TextStyle {
```

Comment:
p2: 구조상 내부에 LineHeight 관련 프로퍼티만 있으니 TextStyle 보다는 LineHeihgt로 네이밍 하는게 좋을 것 같네요!

아니면 요런 구조도 가능하지 않을까요?

```suggestion
enum Font
    struct LineHeight {
       // something ...
    }
    struct Typography {
       // something ...
    }
```

Reply:
수정해서 올리겠습니당 ㅎㅎ